fuzzing/fuzzer_test_methods_test.go

```diff
@@ -80,14 +80,17 @@ func assertFailedTestsExpected(f *fuzzerTestContext, expectFailure bool) {
 // assertCorpusCallSequencesCollected will check to see whether we captured coverage-increasing call sequences in the
 // corpus. It asserts that the actual result matches the provided expected result.
 func assertCorpusCallSequencesCollected(f *fuzzerTestContext, expectCallSequences bool) {
+	// Obtain our count of mutable (often representing just non-reverted coverage increasing) sequences.
+	callSequenceCount := f.fuzzer.corpus.CallSequenceEntryCount(true, false, false)
```

Comment:
Feel like the better choice of function here is `f.fuzzer.corpus.ActiveMutableSequenceCount`

Reply:
Or maybe those two values can differ?